apps/zbugs/vite.config.ts

```diff
@@ -1,5 +1,5 @@
 import react from '@vitejs/plugin-react';
-import {defineConfig, type ViteDevServer} from 'vite';
+import {defineConfig, loadEnv, type ViteDevServer} from 'vite';
```

Comment:
It should not be necessary to load env vars here. See how `DISCORD_WEBHOOK_URL` works? It's not loaded here but is still accessible from `discord.ts`. Just using `process.env` inside the `upload.ts` file should be sufficient.

Also please add these env vars to `.env.example`.